contracts/deployments/eigenlayer-core/11155111.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Note: I don't know what chain this is. You can't add comments to json, but it would be good to include some docs on which chain id is what common name / rpc endpoint / etc